File: python/src/tests/t_acquire_count.py
#!/usr/bin/python3
#
# $Id:$
#
# We test a bit of the atrshmlog here.
#
# This is for the first starter, so only the basic things.

import sys
import atrshmlog

r = atrshmlog.attach()

count = atrshmlog.get_acquire_count()

print('acquire count : ' + str(count) + ' : ')

count = atrshmlog.get_buffer_id()

print('acquire count : ' + str(count) + ' : ')


exit(0);

# end of test

File: perl/src/tests/t_acquire_count.pl
#!/usr/bin/perl
#
# $Id:$
#
# perl test driver

use Atrshmlog;

$result = Atrshmlog::attach();

$count = Atrshmlog::get_acquire_count();

print "aquire count : $count : \n";

$count = Atrshmlog::get_buffer_id();

print "aquire count : $count : \n";

print " \n";

exit (0);

# end of main
